code/datums/components/riding/riding_vehicle.dm

```diff
@@ -123,23 +123,37 @@
 	if(can_slow_down)
 		RegisterSignal(rider, COMSIG_MOVE_INTENT_TOGGLED, PROC_REF(toggle_move_delay))
 		toggle_move_delay(rider)
+		RegisterSignal(rider, COMSIG_ATOM_BULLET_ACT, PROC_REF(check_knockoff))
 
 /datum/component/riding/vehicle/scooter/skateboard/handle_unbuckle(mob/living/rider)
 	. = ..()
 	if(can_slow_down)
 		toggle_move_delay(rider)
 		UnregisterSignal(rider, COMSIG_MOVE_INTENT_TOGGLED)
+	UnregisterSignal(rider, COMSIG_ATOM_BULLET_ACT)
```

Comment:
is this meant to be outside this `if` block?

Reply:
Yes, as it is not applied inside can_slow_down- ok
It is applied in can slow down, it should not be. Will remove it from inside the other if block, thanks for spotting!